src/main/webapp/WEB-INF/tags/instructor/results/responseRow.tag

```diff
@@ -8,87 +8,87 @@
 <%@ attribute name="responseRow" type="teammates.ui.template.InstructorFeedbackResultsResponseRow" required="true" %>
 
 <tr <c:forEach items="${responseRow.rowAttributes.attributes}" var="attr">${attr.key}="${attr.value}"</c:forEach>>
-    <c:if test="${responseRow.giverDisplayed}">
-        <c:if test="${responseRow.giverProfilePictureAColumn}">
-            <td class="middlealign">
-             <c:choose>
-                <c:when test="${not empty responseRow.giverProfilePictureLink}">
-                    <div class="profile-pic-icon-click align-center" data-link="${responseRow.giverProfilePictureLink}">
-                        <a class="student-profile-pic-view-link btn-link">
-                            View Photo
-                        </a>
-                        <img src="" alt="No Image Given" class="hidden">
-                    </div>
-                </c:when>
-                <c:otherwise>
-                        <div class="align-center" data-link="">
-                            <a class="student-profile-pic-view-link btn-link">
-                                No Photo
-                            </a>
-                        </div>
-                </c:otherwise>
-             </c:choose>
-                </td>
-        </c:if>
-        <td class="word-wrap-break middlealign<c:if test="${responseRow.rowGrey}"> color_neutral</c:if>">${fn:escapeXml(responseRow.giverTeam)}</td>
-        <td class="word-wrap-break middlealign<c:if test="${responseRow.rowGrey}"> color_neutral</c:if>">
-        <c:choose>
-            <c:when test="${not empty responseRow.giverProfilePictureLink && !responseRow.giverProfilePictureAColumn}">
-                <div class="profile-pic-icon-hover" data-link="${responseRow.giverProfilePictureLink}">
-                    ${fn:escapeXml(responseRow.giverDisplayableIdentifier)}
-                    <img src="" alt="No Image Given" class="hidden profile-pic-icon-hidden">
-                </div>
-            </c:when>
-            <c:otherwise>
-                ${fn:escapeXml(responseRow.giverDisplayableIdentifier)}
-            </c:otherwise>
-        </c:choose>
-        </td>
+  <c:if test="${responseRow.giverDisplayed}">
+    <c:if test="${responseRow.giverProfilePictureAColumn}">
+      <td class="middlealign">
+       <c:choose>
+        <c:when test="${not empty responseRow.giverProfilePictureLink}">
+          <div class="profile-pic-icon-click align-center" data-link="${responseRow.giverProfilePictureLink}">
+            <a class="student-profile-pic-view-link btn-link">
+              View Photo
+            </a>
+            <img src="" alt="No Image Given" class="hidden">
+          </div>
+        </c:when>
+        <c:otherwise>
+            <div class="align-center" data-link="">
+              <a class="student-profile-pic-view-link btn-link">
+                No Photo
+              </a>
+            </div>
+        </c:otherwise>
+       </c:choose>
```

Comment:
Add an additional space to `<c:choose>` and `</c:choose>`, and 2 additional spaces to everything in between.